.github/workflows/release.yaml

```diff
@@ -68,5 +68,5 @@ jobs:
       env:
         GH_TOKEN: ${{ github.token }}
       run: |
-        GOOS=${{ matrix.goos }} GOARCH=${{ matrix.goarch }} go build -ldflags="-s -w" -o build/pizza-${{ matrix.goos }}-${{ matrix.goarch }}
+        GOOS=${{ matrix.goos }} GOARCH=${{ matrix.goarch }} go build -ldflags="-s -w -X 'github.com/open-sauced/pizza-cli/pkg/utils.Version=1.2.3' -X 'github.com/open-sauced/pizza-cli/pkg/utils.Sha=$(git rev-parse HEAD)'" -o build/pizza-${{ matrix.goos }}-${{ matrix.goarch }}
```

Comment:
```sh
-ldflags="-s -w -X 'github.com/open-sauced/pizza-cli/pkg/utils.Version=1.2.3' -X 'github.com/open-sauced/pizza-cli/pkg/utils.Sha=$(git rev-parse HEAD)'" 
```
The `1.2.3` here will be replaced with `needs.release.outputs.release-tag`?

Reply:
Good catch: yes, that was copy pasted. Thanks @Anush008 😓